mikeio1d/result_network/result_location.py

```diff
@@ -30,12 +30,31 @@ class ResultLocation:
     def __init__(self, data_items, res1d):
         self.data_items = data_items
         self.res1d = res1d
-        self.quantity_label = 'q_'
+        self.quantity_label = "q_"
```

Comment:
It seems that we decide to go for double quotes. I typically prefer in Python to use single quotes, but I saw that in MIKE IO 1D it ended up to be inconsistent. Double quotes it is then ;-).